requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests>=2.12.5
 tqdm>=4.59.0
 ruamel.yaml<=0.16.6
-tabulate
\ No newline at end of file
+tabulate
+numpy==1.22.3
```

Comment:
suggest not including a version, just `numpy` should work for most people.

Reply:
Ok will change that